kuma/javascript/src/payments/__tests__/thank-you.test.jsx

```diff
@@ -0,0 +1,13 @@
+//@flow
```

Comment:
I agree with you in that it feels redundant. I prefer having things in a separate `__test__` folder because I think it's more organized, especially when there's snapshots. And I noticed all the tests had `.test.` like you mentioned, so this was an attempt to have both consistency with the existing test file names and hide the clutter behind a folder. 

Any way, I moved them to /payments and got rid of the `__test__` folder.